org.sonatype.m2e.egit/src/org/sonatype/m2e/egit/internal/EgitScmHandler.java

```diff
@@ -76,9 +99,30 @@ protected int getTimeout() {
     return 30;
   }
 
-  protected URIish getUri(MavenProjectScmInfo info) throws URISyntaxException {
+  protected void runCloneOperation(URIish uri, File location, String refName, SubMonitor pm)
+                  throws InvocationTargetException, IOException, InterruptedException {
+    CloneOperation clone = new CloneOperation(uri, true /* allSelected */, new ArrayList<Ref>(), location, refName,
+          "origin", getTimeout());
+    clone.run(pm.newChild(99));
+
+    fixAutoCRLF(clone.getGitDir());
+  }
+
+  protected boolean onAuthFailed() {
+    final boolean[] result = new boolean[]{false};
+    Display.getDefault().syncExec(new Runnable() {
+      @Override
+      public void run() {
```

Comment:
`The method run() of type new Runnable(){} must override a superclass method.`

This warning shows up because the project BREE is J2SE-1.5 and I happen to have a Java 5 JDK, exhibiting the issue.

We should bump the BREE to 1.7, as well as the project version to 0.15.0-SNAPSHOT